scheduler/src/scheduler.rs

```diff
@@ -314,6 +307,14 @@ impl Scheduler {
         }
     }
 
+    fn abort(&self, client: u32) -> Result<(), Error> {
+        tracing::warn!("aborting client {}", client);
```

Comment:
can we move this to line number 314? so the warning message is valid at that point..